css-view-transitions-1/Overview.bs

```diff
@@ -1048,6 +1048,12 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			Note: This is used to detect changes in the [=snapshot containing block size=],
 			which causes the transition to [=skip the view transition|skip=].
 			[Discussion of this behavior](https://github.com/w3c/csswg-drafts/issues/8045).
+
+		: <dfn>old state captured steps</dfn>
+		:: An algorithm accepting nothing, or null.
+			Initially null.
```

Comment:
Can you add normative text saying only one of "update callback" or "old state captured steps" should be set?

And a note to clarify that the callback is optionally set for same-doc transitions and "old state captured steps" is always set if this is on the old doc for a cross-document transition.

Reply:
Done